packages/index.ts

```diff
@@ -1,3 +1,7 @@
+import { injectGlobalCss } from "./shared/styles/global";
+
 export { Badge } from "./badge";
 export { BadgeButton } from "./badge";
 export { Column, Table, Cell, TextCell, HeaderCell } from "./table";
+
+injectGlobalCss();
```

Comment:
We should discuss this. As this will only work if we import from `@dcos/ui-kit` and not with submodule imports.